### ISSUE TITLE
Documentation update suggestion for top-level README.md

### ISSUE THREAD

**Issue by marktomlinson1** [terraform-google-modules/terraform-example-foundation#610]
### TL;DR

Folder and project structure README.md documentation for step 1-org would benefit from update, removing the two networking projects which are created in a later step. 

### Expected behavior

Project structure under 1-org should be

example-organization
└── fldr-common
    ├── prj-c-logging
    ├── prj-c-billing-logs
    ├── prj-c-dns-hub
    ├── prj-c-interconnect
    ├── prj-c-scc
    └── prj-c-secrets

### Observed behavior

Project structure under 1-org is

example-organization
└── fldr-common
    ├── prj-c-logging
    ├── prj-c-base-net-hub
    ├── prj-c-billing-logs
    ├── prj-c-dns-hub
    ├── prj-c-interconnect
    ├── prj-c-restricted-net-hub
    ├── prj-c-scc
    └── prj-c-secrets

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
0.13
```

### Additional information

Running Terraform locally in Cloud Shell
